worlds/sc2/Regions.py

```diff
@@ -606,8 +606,6 @@ def setup_final_location(final_location, location_cache):
     # Final location should be near the end of the cache
     for i in range(len(location_cache) - 1, -1, -1):
         if location_cache[i].name == final_location:
-            location_cache[i].locked = True
-            location_cache[i].event = True
```

Comment:
I think this is still important, as these are core AP flags that let us put the victory flag somewhere? I'm not totally sure as I'm not too familiar with the core code.